src/Chapter2.hs

```diff
@@ -630,11 +631,11 @@ Write a function that takes elements of a list only in even positions.
 -}
 takeEven :: [a] -> [a]
 takeEven [] = []
-takeEven y = go 0 y
+takeEven y = go True y
   where
-    go :: Int -> [a] -> [a]
+    go :: Bool -> [a] -> [a]
     go _ [] = [] 
-    go acc (x:xs) = if (mod acc 2) == 0 then x : go (acc + 1) xs else go (acc + 1) xs
+    go even (x:xs) = if even then x : go (not even) xs else go (not even) xs
```

Comment:
`even` is a standard function in Haskell (that's why you can see the warning below). It works because Haskell allows shadowing of variables locally. But generally, it's better to avoid name collisions for readability. So, the local variable could be named `isEven`

Reply:
Got it, I changed it to avoid using the standard